src/Map/assets/src/abstract_map_controller.ts

```diff
@@ -111,12 +111,14 @@ export default abstract class<
     protected createInfoWindow({
         definition,
         marker,
+        onMarkerClick = false,
```

Comment:
Not sure this should be exposed like this, seems to complexify the contract, while not beeing this "explanatory"